vsce/rush-command-webview/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
```

Comment:
The `vsce` acronym may be unfamiliar to most people. After chatting, let's rename the folder to `vscode-extensions`